SetupDataPkg/Tools/ConfigEditor.py

```diff
@@ -955,20 +899,17 @@ def get_save_file_name(self, extension):
             return None
 
     def save_delta_file(self, full=False):
-        path = self.get_save_file_name("dlt csv")
+        path = self.get_save_file_name("csv")
         if not path:
             return
 
-        # assumption is there is one yaml file and one xml file
-        # yml changes get saved to dlt, xml changes get saved to csv
+        # assumption is there may be multiple xml file
+        # xml changes get saved to csv
         for file_id in self.cfg_data_list:
             self.update_config_data_on_page()
 
-            if self.cfg_data_list[file_id].config_type == 'yml':
-                dlt_path = path
-            else:
-                # replace .dlt with .csv and append unique filename identifier
-                dlt_path = path[:-4] + self.cfg_data_list[file_id].cfg_data_obj._cur_page + ".csv"
+            # replace .dlt with .csv and append unique filename identifier
+            dlt_path = path[:-4] + self.cfg_data_list[file_id].cfg_data_obj._cur_page + ".csv"

```

Comment:
This should no longer be needed, the [:-4] as we don't have .dlt at the end anymore. Also, we can change the variable name to csv_path.

Reply:
But we should test and confirm .dlt isn't getting added anymore.

Reply:
Will do more testing before removing this.

Reply:
The variable name is updated. But the path logic is left as is since the unique identifier is before the `.csv` suffix. Can clean this file further in a designated PR.